libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -97,6 +97,14 @@ AC_Fence::AC_Fence(const AP_AHRS& ahrs, const AP_InertialNav& inav) :
     }
 }
 
+void AC_Fence::enable(bool true_false)
+{
+    _enabled = true_false;
+    if (!true_false) {
```

Comment:
We should change the argument name, this line is ugly. Something like `enable` is much more readable.